src/app/clusters/basic-information/BasicInformationCluster.cpp

```diff
@@ -1,6 +1,6 @@
-/**
- *
- *    Copyright (c) 2020-2022 Project CHIP Authors
+/*
+ *    Copyright (c) 2025 Project CHIP Authors
```

Comment:
```suggestion
 *    Copyright (c) 2020-2025 Project CHIP Authors
```
Given that the file was just renamed and a bunch of code kept.